### ISSUE TITLE
[PropertyFieldOrder] When items is an empty array, component renders the text "0"

### ISSUE THREAD

**Issue by TheTedAdams** [SharePoint/sp-dev-fx-property-controls#152]
#### Category
- [ ] Enhancement
- [x] Bug
- [ ] Question

#### Version

1.14.0

#### Expected / Desired Behavior / Question
When passing an empty array to items, the control should render nothing under the label (Or, even nicer, some text stating there are no items)

#### Observed Behavior
When the items array is empty, the control renders the text "0" under the label.

#### Steps to Reproduce
Add the control and pass an empty array to items. Observe that the control renders the text "0" where the items would go.

#### Suggested Resolution
This is another issue of inappropriate use of a truthy/falsey check.

[Relevant Code:](https://github.com/SharePoint/sp-dev-fx-property-controls/blob/master/src/propertyFields/order/PropertyFieldOrderHost.tsx#L66)
```js
(this.state.items && this.state.items.length) && <div
```

The appropriate check is actually made above on line 52. I remember react used to not render "0" when passed the length of an empty array, so I'm guessing this was written back then and just hasn't been updated.

**Comment by TheTedAdams** [comment]
Just to add one clarification to this, because the obvious question is why am I rendering this control when items is empty?

For context, I am using this control in combination with the PropertyFieldMultiSelect to allow the user to add and remove items from the array, and just beneath that control the order of items in the array.

The problem is because this control has no hidden param, to hide it I have to completely exclude it from the groupFields. When I do that I was seeing the pane get re-rendered each time I added or removed an item in the PropertyFieldMultiSelect, which caused the MultiSelect control to close each time an item was added or removed. Very annoying.
